src/vs/platform/markers/common/problemMatcher.ts

```diff
@@ -415,6 +441,13 @@ export namespace Config {
 		*/
 		regexp?: string;
 
+		/**
+		 * Whether the pattern matches a whole file, or a location (file/line)
+		 *
```

Comment:
Can we add a comment to say that the value is only valid on the first entry. Like we do with `loop`.